### DISCUSSION TITLE
"Modify Content" keeps creating homunculi

### DISCUSSION THREAD

**Discussion by jaimeastorga2000** [lllyasviel/Fooocus#3283] [Q&A] [answered]
Improve Detail works great, Outpaint works great, but whenever I try to Modify Content I more often than not end up with a little copy of the original instead of the object I wanted added.

Here's an example. Earlier today I generated the following image of Harry James Potter-Evans-Verres:

![2024-07-14_16-49-46_7396](https://github.com/user-attachments/assets/9472b02a-ea6c-48a2-b7ff-11e0007156b8)

I wanted to inpaint a lightning bolt scar to his forehead, so I use the image as an input and try to Modify Content with the prompt "scar shaped like a lightning bolt".

_This_ is what comes out:

![2024-07-14_17-06-54_8301](https://github.com/user-attachments/assets/d5f04f3a-db53-4c1b-b903-5cac3e88d759)

![2024-07-14_17-09-00_4876](https://github.com/user-attachments/assets/e6f007e4-ff7d-4e26-bc22-d501789c80b7)

It's horrifying.

Later today I decided to try genning Amaryllis Penndraig:

![2024-07-14_20-42-22_2739](https://github.com/user-attachments/assets/e458b576-0d47-467e-9686-768377d7e9c1)

Looks good, except her armor is supposed to have an stylized logo of an anvil on it. No problem, I'll just fire up the Modify Content setting, add "anvil logo" to the additional prompt, an -- _mother of God!_

![2024-07-14_20-46-27_3931](https://github.com/user-attachments/assets/4d158624-9574-46c0-becc-4b9cf5d97623)

![2024-07-14_20-48-04_5798](https://github.com/user-attachments/assets/caaaff5e-e710-4fec-ac4f-2a0a02da60c0)

So, yeah. Where did I screw up and how do I stop this from happening?

**Comment by IPv6** [comment]
Do you use Image Prompt or something from the controlnets tab? In that case image used for controlnet should present masked area only, not a whole image

**Comment by mashb1t** [comment, marked as answer]
When using `Modify Content` you have to adjust all your prompts to what you want to actually inpaint, because if you don't it will inpaint the exact thing in your prompt again.
This includes image prompts, LoRAs, positive and negative prompts etc.
